### PR TITLE
Add `cargo ci` alias for `ci` tool

### LINKED ISSUE #13197: Create alias to `ci` tool
## What problem does this solve or what need does it fill?

Writing `cargo run -p ci -- ...` is lengthy and repetitive.

## What solution would you like?

Add an alias, `cargo ci`, which would be equivalent. This can be added to `.cargo/config.toml`.

```toml
[alias]
ci = "run --package ci --"
```

Unfortunately there are a few issues with this. `config.toml` is currently `.gitignore`-d because we encourage developers to configure their own, faster linkers. By adding the `[alias]` table for everyone, we would have to remove `config.toml` from `.gitignore`, making every local modification to it visible in Git's staged changes.

Even if we had a `config.toml` with just the `[alias]` table and `config_fast_builds.toml` separate, it would still run into the same issue because developers would have to copy the contents of `config_fast_builds.toml` over, thus showing up in Git.

I'm not sure how to solve this, so I'm creating an issue for it. I think if it were possible [to merge multiple `config.toml`](https://doc.rust-lang.org/cargo/reference/config.html#hierarchical-structure) files, this would be perfect, but I can't think of anything right now.

## What alternative(s) have you considered?

- Add this directly to `config_fast_builds.toml`, so you must opt-in to use the alias.
- Do not implement this feature.

## Additional context

- Inspired by [`cargo-xtask`](https://github.com/matklad/cargo-xtask).
- [Example usage](https://github.com/rust-lang/cargo/blob/54d8815d04fa3816edc207bbc4dd36bf18014dbc/.cargo/config.toml#L2-L4), in Cargo itself.
- [Reference documentation](https://doc.rust-lang.org/cargo/reference/config.html#alias) for `[alias]`.

### PR DESCRIPTION
**Pull request by BD103**
# Objective

- Running `cargo run --package ci -- ...` is lengthy, and `cargo r -p ci -- ...` isn't much better.
- Closes #13197.

## Solution

- Add an opt-in alias to the `ci` tool with `cargo ci ...`.
  - You need to copy `config_fast_builds.toml` to `config.toml` to enable this functionality.

## Testing

- Copy `config_fast_builds.toml` to `config.toml`, then run `cargo ci --help`.

---

## Changelog

- Added `cargo ci` alias for internal `ci` tool.

### CODE REVIEW

**Comment by BD103** [comment]
It's unfortunate that this is opt-in, but this is the least-friction solution. The alternative probably would involve:

- Renaming `config_fast_builds.toml` to `config.toml`, breaking a few links on the website.
- Remove `config.toml` from `.gitignore`.
- Disable all of the fast-build shenanigans by default.

The issue with that is that all changes to `config.toml`, such as enabling fast builds, would be tracked by Git. I feel like that would interfere with the development workflow too much that it does not justify the change.

**Comment by Brezak** [comment]
Given #13279. Could we add a setup command to our CI tool that asks the contributor some questions and then setups the `config.toml` for them?

**Comment by BD103** [comment]
That sounds like a great follow-up PR! I'll create an issue for it.

**Review by Brezak — APPROVED** [review]

File: .cargo/config_fast_builds.toml [hunk 1]
```diff
@@ -144,3 +144,8 @@ rustflags = [
 # In most cases the gains are negligible, but if you are on macos and have slow compile times you should see significant gains.
 # [profile.dev]
 # debug = 1
+
+# This is enables you to run the CI tool using `cargo ci`.
+# This is not enabled by default, you need to copy this file to `config.toml`.
+[alias]
+ci = "run --package ci --"
```
Comment (Brezak) [inline .cargo/config_fast_builds.toml:151]: You're missing a new line.

Comment (BD103) [reply .cargo/config_fast_builds.toml:151]: I don't think so, it shows up in my editor.

image

I think Github hides newlines at the end of the file.

**Review by Brezak — APPROVED** [review]

**Review by bushrat011899 — APPROVED** [review]

**Pull request merged**